.github/workflows/kubernetes-deploy.yaml

```diff
@@ -31,6 +31,11 @@ jobs:
           docker build -t ray:test --build-arg TARGETARCH="amd64" -f ./Dockerfile-ray-node .
           kind load docker-image ray:test
           docker image rm ray:test
+      - name: Build and load proxy
+        run: |
+          docker build -t proxy:test --build-arg TARGETARCH="amd64" -f ./proxy/Dockerfile .
+          kind load docker-image proxy:test
```

Comment:
are using kind just for testing here, Aki?

Reply:
Yes, the proxy image must be loaded into kind for this test.